src/googleCloud/DatasetSelector.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DicomStorePicker from './DicomStorePicker';
+import DatasetPicker from './DatasetPicker';
+import ProjectPicker from './ProjectPicker';
+import LocationPicker from './LocationPicker';
+import './googleCloud.css';
+
+export default class DatasetSelector extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      project: null,
+      location: null,
+      dataset: null,
+      unloading: false,
+    };
+
+    this.onProjectSelect = this.onProjectSelect.bind(this);
```

Comment:
You can avoid all of these `.bind` calls by changing the class property functions below to be arrow functions: 

```javascript
onProjectSelect = (project) => {
...
}
```

Reply:
done